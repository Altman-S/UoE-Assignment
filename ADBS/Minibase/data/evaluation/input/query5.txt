Q(x, y, z, u, w, t) :- R(x, y, z), S(u, w, t), x = u