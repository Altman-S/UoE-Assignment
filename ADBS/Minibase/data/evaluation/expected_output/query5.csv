1, 9, 'adbs', 1, 'smith', 8
2, 7, 'anlp', 2, 'anka', 2
4, 2, 'ids', 4, 'rhcp', 3
5, 0, 'sdp', 5, 'bowie', 4
8, 1, 'mlpr', 8, 'pistols', 5
8, 9, 'rl', 8, 'pistols', 5
8, 9, 'ppls', 8, 'pistols', 5